site/content/3.12/arangograph/deployments/_index.md

```diff
@@ -293,10 +293,26 @@ This operation is **irreversible**. Please proceed with caution.
    holds the deployment you wish to delete.
 2. In the __Deployments__ page, click the deployment you wish to delete.
 3. Click the __Delete/Lock__ entry in the navigation.
-4. Click the __Delete deployment__ button.
+4. Click the __Delete deployment...__ button.
 5. In the modal dialog, confirm the deletion by entering `Delete!` into the
    designated text field.
 6. Confirm the deletion by pressing the __Yes__ button.
 7. You will be taken back to the deployments page of the project.
    The deployment being deleted will display the __Deleting__ status until it has
    been successfully removed.
+
+{{< info >}}
+Locked deployments cannot be deleted. Learn more about
+[locked resources](../security-and-access-control/_index.md#locked-resources) in ArangoGraph.
+{{< /info >}}
+
+## How to pause a deployment
+
+1. In the main navigation, in the __Projects__ section, click the project that
```

Comment:
Instead of going via Projects, ee could describe the flow: click Deployments and then the name of the deployment you wish to delete, then we don't have to use the fragment "the deployment you wish to delete" twice

Reply:
Yes, we should remove the first step via Projects, same for deleting a deployment. IIRC this was necessary in the past, when you couldn't access Deployments straight away but had to go through Projects first.